### PR TITLE
Unite plugin remvoe list

### PR DESCRIPTION
**Pull request by gadisn**

### CODE REVIEW

**Comment by bd82** [comment]
you need to sign the CLA

File: resources/tests/serviceTestFramework.js [hunk 1]
```diff
@@ -38,15 +38,8 @@ define("STF", [
...
-    remove: [
-      "sap.watt.saptoolsets.mta.serverplatform.neo.hcpprojectspace",
-      "sap.watt.saptoolsets.mta.common.deployToCF",
-      "sap.watt.saptoolsets.templates.common.hanatemplates",
-      "sap.watt.ideplatform.orion.orionbackend",
-      "sap.watt.saptoolsets.orionmigration",
-      /^sap\.watt\.ideplatform\.che/
-    ],
-    add: fakePlugins.defaultFakePlugins(true)
+    remove: mockConf.pluginsToRemove(),
```
Comment (bd82) [inline resources/tests/serviceTestFramework.js]: The plugins to remove and those to add are connected/related.
So it makes more sense to have a single function that returns an **object**
with remove and add properties.

That way the mockConf can support different variations of mocks in the future.
```javascript
mockConf.getMocksForInMemory()
mockConf.getMocksForCF()
mockConfig.getMocksForXXX()
```

Comment (gadisn) [reply resources/tests/serviceTestFramework.js]: @bd82 Done - can you have another look?

**Comment by bd82** [comment]
you need to fixed your email used in the commit

> Gadi Sneh seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please add the email address used for this commit to your account.

otherwise you cannot sign the CLA

**Pull request merged**